### ISSUE TITLE
Severall Issues

### ISSUE THREAD

**Issue by aevernum** [folkswithhats/fedy#571]
Hey! Got several issues

> Gjs-Message: JS WARNING: [./app.js 535]: reference to undefined property "license"
Last metadata expiration check: 0:56:39 ago on Thu 11 Jan 2018 06:43:00 PM EET.
google-chrome-stable_current_x86_64.rpm   

> Gjs-Message: JS WARNING: [./app.js 404]: Too many arguments to function GLib.timeout_add: expected 3, got 4
google-chrome  

> Gjs-Message: JS WARNING: [./app.js 404]: Too many arguments to function GLib.timeout_add: expected 3, got 4
Last metadata expiration check: 0:02:49 ago on Thu 11 Jan 2018 07:41:10 PM EET.
teamviewer.i686.rpm  

> Gjs-Message: JS WARNING: [./app.js 404]: Too many arguments to function GLib.timeout_add: expected 3, got 4
--2018-01-11 19:46:37--  https://github.com/atom/atom/releases/download/v1.23.3/atom.x86_64.rpm

> Gjs-Message: JS WARNING: [./app.js 404]: Too many arguments to function GLib.timeout_add: expected 3, got 4
Repository successfully enabled.
Copr repo for Arc owned by user501254   

got this ->  **JS WARNING: [./app.js 404]** in other installations. It`s non-critical, just annoyin :)

**Comment by bddenhartog** [comment]
It looks like the `timeout_add()` function may have changed.

Do you have a comprehensive list of the plugins you were installing when you encountered the 404s?

**Comment by aevernum** [comment]
No, sorry. On Sunday I will have an installation on a clean PC, I'll try to prepare.
For now, I can only say that this is caused by such plugins:
> Arc-icon-theme
Arc-theme (with transparency)
Atom
Google Chrome
Telegram
Numix Themes

I did not install the rest, but there is a suspicion :)

**Comment by sudoforge** [comment]
> In any case, after I brought the function to three arguments, I received another warning:
> 
>     Gjs-Message: JS WARNING: [./app.js 535]: reference to undefined property "license"
> 
> It should be a plugin, but missing in plugins folder or not described?

I've created a separate issue for this: #602
